components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2203,6 +2206,14 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         await this.installationAdminDb.setSettings(newSettings);
     }
 
+    async adminGetTelemetryData(ctx: TraceContext): Promise<Data> {
+        traceAPIParams(ctx, {});
+
+       return await this.telemetryDataProvider.getTelemetryData();
+    }
```

Comment:
We should add a guard that only admins can access this endpoint. Just to make sure, this information is not publicly available for all Gitpod instances.

```suggestion
    async adminGetTelemetryData(ctx: TraceContext): Promise<Data> {
       traceAPIParams(ctx, {});
       await this.guardAdminAccess("adminGetTelemetryData", {}, Permission.ADMIN_API);
       return await this.telemetryDataProvider.getTelemetryData();
    }
```